src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -69,6 +69,10 @@
 
   private static final Logger logger = Logger.getLogger(DeclarationGenerator.class.getName());
   private static final String INTERNAL_NAMESPACE = "ಠ_ಠ.cl2dts_internal";
+
+  // Comments in .d.ts and .js golden files starting with '//!!' are stripped.
+  public static final String GOLDEN_FILE_COMMENTS_REGEXP = "(?m)^\\s*//!!.*\\n";
```

Comment:
more idiomatic (and performant) to hold a Pattern object here
public static final Pattern BLAH = Pattern.compile(string)